projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-user-create-endpoint.php

```diff
@@ -76,7 +77,7 @@ public function create_or_get_user() {
 			$this->user_data->url          = isset( $this->user_data->URL ) ? $this->user_data->URL : '';
 			$this->user_data->display_name = $this->user_data->name;
 			$this->user_data->description  = '';
-			$user                          = Jetpack_SSO_Helpers::generate_user( $this->user_data );
+			$user                          = Helpers::generate_user( $this->user_data );
```

Comment:
You make a good point. I'm not sure where I would extract it though, so it's available in both the Jetpack plugin and the Connection package. Do you have any suggestions? Should it be in `Automattic\Jetpack\Connection\Utils`? It feels off since it's not strictly connection-related, but at the same time it does create a user with attached WordPress.com info.

Reply:
Yeah, it does look out of place whenever we put it.
I can't think of anything better than `Utils`.

Reply:
Done in d1571de8f8424a76bded6eabb26410826eea3745